tools/functional-tester/etcd-runner/command/root.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2017 The etcd Authors
```

Comment:
I am following the convention from cobra doc. https://github.com/spf13/cobra#manually-create-rootcmd.

let the rootcmd to live in the same pkg as rest of cmds allows a simpler way of sharing global variable, or else the sub cmd needs to access the etcd-runner pkg to get it, https://github.com/coreos/etcd/blob/master/tools/functional-tester/etcd-runner/main.go#L34.